packages/react/src/index.ts

```diff
@@ -10,8 +10,15 @@ import type {
   WalletState,
   ConnectedChain
 } from '@web3-onboard/core'
-import type { Chain } from '@web3-onboard/common'
-import type { AppState } from '@web3-onboard/core/dist/types'
+import type { Chain, WalletInit } from '@web3-onboard/common'
+import type {
+  AccountCenter,
+  AppState,
+  CustomNotification,
+  Notification,
+  NotifyOptions,
+  UpdateNotification
+} from '@web3-onboard/core/dist/types'
```

Comment:
@Adamj1232 We shouldn't have to do this right? 
I feel that these should be exported at the root (`index.ts`) level so that we can do this instead:

```suggestion
} from '@web3-onboard/core'
```